pkg/verification/service/verification_service_test.go

```diff
@@ -12,6 +12,8 @@ import (
 	"testing"
 	"time"
 
+	sender2 "github.com/codeready-toolchain/registration-service/pkg/verification/sender"
```

Comment:
nit: think it'd be better to rename this to vsender or something

Reply:
renamed to senderpkg